### PR TITLE
Ada.Strings module

### PR DESCRIPTION
**Pull request by frank-at-adacore**

### CODE REVIEW

File: courses/fundamentals_of_ada/labs/answers/spec_892_ada_strings.txt [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+      First : Natural;
+      Last  : Natural;
+   begin
+      First := Ada.Strings.Fixed.Index (Str, """");
+      if First in Str'range then
+         Last := Ada.Strings.Fixed.Index
+             (Source  => Str (First + 1 .. Str'last),
+              Pattern => """");
+         if Last in Str'range then
+            return Ada.Strings.Fixed.Replace_Slice (Str,
```
Comment (leogermond) [inline courses/fundamentals_of_ada/labs/answers/spec_892_ada_strings.txt:36]: Multiline args indent style is incorrect, result is weird, should go to the line before the opening parenthesis

**Review by leogermond — CHANGES_REQUESTED** [review]

File: courses/fundamentals_of_ada/labs/answers/spec_892_ada_strings.txt (hunk 1 above)
Comment (frank-at-adacore) [reply courses/fundamentals_of_ada/labs/answers/spec_892_ada_strings.txt:36]: Not sure what you mean. Line 35 should be lined up with Line 32 (and it is)

Comment (leogermond) [reply courses/fundamentals_of_ada/labs/answers/spec_892_ada_strings.txt:36]: line 36 and 37, the columns are crushed to the right

Comment (frank-at-adacore) [reply courses/fundamentals_of_ada/labs/answers/spec_892_ada_strings.txt:36]: This is what "gnatpp" does in a return (I guess) - I didn't make it look like that :)
That being said, I will clean it u

**Review by leogermond — APPROVED** [review]

**Pull request merged**